Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -331,6 +345,81 @@ public override int GetHashCode()
         {
             return base.GetHashCode();
         }
+
+        /// <summary>
+        /// Determines if the local user owns this Session.
+        /// This information can be used to gate some Session modifying actions.
+        /// </summary>
+        /// <returns>
+        /// True if the Session is owned by the local user.
+        /// False if either it was determined that the user doesn't own this Session,
+        /// or if the operation to query that information failed.
+        /// </returns>
+        public bool IsLocalUserOwnerOfSession
+        {
+            get
+            {
+                ProductUserId localUserId = EOSManager.Instance?.GetProductUserId();
+
+                // If we can't even get the local user's id, that state suggests they shouldn't be owning any Sessions
+                if (localUserId == null)
+                {
+                    return false;
+                }
+
+                // If the ActiveSession isn't set, then this user isn't a part of the Session
+                // Can't possibly own it
+                if (ActiveSession == null)
+                {
+                    return false;
+                }
+
+                ActiveSessionCopyInfoOptions options = new ActiveSessionCopyInfoOptions() { };
+                Result copyResult = ActiveSession.CopyInfo(ref options, out ActiveSessionInfo? sessionInfo);
+
+                if (copyResult != Result.Success || !sessionInfo.HasValue || !sessionInfo.Value.SessionDetails.HasValue)
+                {
+                    return false;
+                }
+
+                return sessionInfo.Value.SessionDetails.Value.OwnerUserId.Equals(localUserId);
+            }
+        }
+
+        public bool TryGetRegisteredUsers(out List<ProductUserId> userIds)
+        {
+            if (ActiveSession == null)
+            {
+                userIds = null;
+                return false;
+            }
```

Comment:
You can simplify several of the conditionals within this function if you start the function with:
```csharp
userIds = null;
```

Reply:
Oh, huh. I never considered doing that for out parameter things. This would remove some redundancy.